cms/templates/js/add-xblock-component-menu-problem.underscore

```diff
@@ -4,9 +4,6 @@
             <a class="link-tab" href="#tab1"><%= gettext("Common Problem Types") %></a>
         </li>
         <li>
-            <a class="link-tab" href="#tab2"><%= gettext("Common Problems with Hints and Feedback") %></a>
```

Comment:
I'm not sure. I also didn't realize that T&L didn't provide anything more than test review. The reviewers were me and Will Daly, but neither of us have extensive Studio experience.